config.js

```diff
@@ -1119,11 +1120,12 @@ function _get_config_root() {
 */
 function _set_nc_config_to_env() {
     const config_to_env = ['NOOBAA_LOG_LEVEL', 'UV_THREADPOOL_SIZE', 'GPFS_DL_PATH', 'NSFS_ENABLE_DYNAMIC_SUPPLEMENTAL_GROUPS'];
-    Object.values(config_to_env).forEach(function(key) {
-        if (config[key] !== undefined) {
-            process.env[key] = config[key];
+    for (const configuration_key of config_to_env) {
+        if (config && Object.keys(config).includes(configuration_key) && config[configuration_key] !== undefined) {
```

Comment:
Is there a case where we don't have `config`? (i want to understand that addition `if (config &&`).

Reply:
this was not the issue, I added it just in case